docs/en/setup/Configuration-File.md

```diff
@@ -150,6 +152,12 @@ trigger:
 
 The Trigger executed successfully at least once, after success, the next stage could be continued. Otherwise, there is an error and exit.
 
+If you get the following error message when using the trigger, You should check whether the port is useful. A possible reason may be that the exporting service needs time to set up, so you can add a larger interval to avoid the error.
```

Comment:
Did you use the `wait` setting to wait for the resource ready?

Reply:
Yes, the resouce is ready before port-forward, I can use `curl` to get data.

Reply:
Yes, the resouce is ready before port-forward, I can use curl to get data.

Reply:
Does the e2e framework support `NodePort` or `LoadBalancer`? They are the more canonical ways to expose services than `port-forward`.

Reply:
@kezhenxu94 How does the `swctl` connect to UI in current e2e cases?

Reply:
We have settings to expose the ports in cluster

https://github.com/apache/skywalking-infra-e2e/blob/5adf908561e2c93e8ee0d82e6eb549cd4cb3b234/docs/en/setup/Configuration-File.md?plain=1#L43-L46

And you can reference via `${<resourceType>_<resourceName>_host}:${<resourceType>_<resourceName>_<port>}`, such as `${service_skywalking_ui_host}:${service_skywalking_ui_80}`

See example https://github.com/apache/skywalking/blob/7aa0ba70483b621d40452b0b2f53423f59f039e5/test/e2e-v2/cases/istio/als/e2e.yaml#L26-L29

Reply:
Thanks @zhenxu

And FWIK, the item is not special to the e2e framework. The user should ensure a service accessible before accessing it. It's common sense more than a FAQ which the e2e user should care about.